docs/BUILD.bazel

```diff
@@ -25,7 +25,83 @@ genrule(
             "theme/da_theme/**",
         ],
     ) + [
+        "@npm//acorn",
```

Comment:
Could you add a comment pointing to [this issue](https://github.com/bazelbuild/rules_nodejs/issues/1553) for future reference?

Reply:
added a comment.